sim/rogue/expose_armor.go

```diff
@@ -51,7 +43,7 @@ func (rogue *Rogue) registerExposeArmorSpell() {
 			result := spell.CalcOutcome(sim, target, spell.OutcomeMeleeSpecialHit)
 			if result.Landed() {
 				debuffAura := rogue.ExposeArmorAuras.Get(target)
-				debuffAura.Duration = rogue.exposeArmorDurations[rogue.ComboPoints()]
+				debuffAura.Duration = time.Second * 30
```

Comment:
This can be set once on initialization instead of on every apply if its constant